pnpm-lock.yaml

```diff
@@ -32157,6 +32167,15 @@ packages:
       - supports-color
     dev: true
 
+  /mdast-util-heading-range@4.0.0:
```

Comment:
Another kind of unrelated change?  Or is it related to your updates to build tools?